lib/public/Model.js

```diff
@@ -58,6 +61,8 @@ export default class Model extends Observable {
      * @returns {vnode} The page to be loaded
      */
     handleLocationChange() {
+        // Clear the editor before navigating
+        this.logs.clearEditor();
```

Comment:
Shouldn't this be in a onremove method?

Reply:
Calling the onremove like this: 
![image](https://user-images.githubusercontent.com/14125058/83862216-ebcc1000-a721-11ea-94de-879fb3a52d87.png)
Doesn't clear the editor (it's never called upon

Reply:
Then it is most likely on the wrong component as the onremove is called in the tag overview/detail and in logs (see the open PR).

Reply:
Fixed :)

Reply:
@mvegter